src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/tools/GoldPan.java

```diff
@@ -61,12 +61,16 @@ public GoldPan(ItemGroup itemGroup, SlimefunItemStack item, RecipeType recipeTyp
     }
 
     /**
-     * This method returns the target {@link Material} for this {@link GoldPan}.
+     * This method returns the target {@link Material}s for this {@link GoldPan}.
      * 
      * @return The {@link Material} this {@link GoldPan} can be used on
      */
-    public @Nonnull Material getInputMaterial() {
-        return Material.GRAVEL;
+    public @Nonnull Set<Material> getInputMaterials() {
+        Set<Material> materials = new HashSet<>();
+
+        materials.add(Material.GRAVEL);
+
+        return materials;
     }
```

Comment:
It is not very efficient to create a new `HashSet` here everytime, make a private final field and just return that one here.